src/envs/pybullet_env.py

```diff
@@ -26,9 +25,7 @@ class PyBulletEnv(BaseEnv):
     # General robot parameters.
     _max_vel_norm: ClassVar[float] = 0.05
     _grasp_tol: ClassVar[float] = 0.05
-    _move_to_pose_tol: ClassVar[float] = 0.0001
```

Comment:
have you checked that these are the only paramters that need to be moved?

Reply:
yep